docs/data-access/data-access.md

```diff
@@ -4,33 +4,44 @@ title: Applying for Access to Controlled Data
 sidebar_label: Applying for Access to Controlled Data
 ---
 
-The **Data Access Compliance Office (DACO)** handles approval for accessing controlled molecular data in the ARGO Data Platform.
+The **Data Access Compliance Office (DACO)** handles requests and approvals for accessing controlled molecular data in the ARGO Data Platform.
```

Comment:
Should we use `ICGC-ARGO` instead of just `ARGO` (since we use it on the main page too)?